pkg/controller/v1beta1/inferenceservice/reconcilers/autoscaler/autoscaler_reconciler.go

```diff
@@ -70,27 +81,21 @@ func getAutoscalerClass(metadata metav1.ObjectMeta) constants.AutoscalerClassTyp
 
 func createAutoscaler(client client.Client,
 	scheme *runtime.Scheme, componentMeta metav1.ObjectMeta,
-	componentExt *v1beta1.ComponentExtensionSpec) (*Autoscaler, error) {
-	as := &Autoscaler{}
+	componentExt *v1beta1.ComponentExtensionSpec) (Autoscaler, error) {
 	ac := getAutoscalerClass(componentMeta)
```

Comment:
We probably need a configuration for the default autoscaler type in addition to the annotation overrides